packages/components/Button/src/__snapshots__/Button.test.tsx.snap

```diff
@@ -27,6 +27,7 @@ exports[`Button default 1`] = `
       }
     }
     accessible={true}
+    collapsable={false}
```

Comment:
Note to self: check it we should enable this prop for performance gains. I vaguely recall this might do something similar to compose, where less views are in the render tree.

Reply:
yep, also similar to react fragments